motionplan/cBiRRT.go

```diff
@@ -240,22 +287,27 @@ func (mp *cBiRRTMotionPlanner) sample(algOpts *cbirrtOptions, rSeed node, sample
 func (mp *cBiRRTMotionPlanner) constrainedExtend(
 	ctx context.Context,
 	algOpts *cbirrtOptions,
+	randseed *rand.Rand,
 	rrtMap map[node]node,
 	near, target node,
-) node {
+	mchan chan node,
+) {
 	oldNear := near
-	for i := 0; true; i++ {
+	for i := 0; i < 1000; i++ {
```

Comment:
[nit] magic number, we should add a comment explaining what this loop is doing at the very least